src/main/webapp/WEB-INF/tags/instructor/home/sort.tag

```diff
@@ -28,5 +28,9 @@
                 </label>
             </div>
         </div>
+        <div class="col-md-3">
+            <a class="btn btn-primary btn-md pull-right" href="${data.instructorCourseLink}" 
+                id="addNewCourse">Add New Course</a>
+        </div>
```

Comment:
Yupp, something like that, can refer to the screenshot above, now that the search bar encompasses the whole screen from left to right (the well that is used in every search bar encompasses from left to right), there is no place to put the button in a place that doesn't break the look.

Currently, the only way to make it in line with the rest of the page is to insert it here, perhaps there is a better place to put this or to rethink whether this button should be there since it directs to Courses page? We could open a new issue for that, todo when JSTL is done maybe?